guides/ansible_engine/3-variables/README.md

```diff
@@ -164,7 +163,6 @@ handlers:
 Your new, improved playbook is done! Let's take a second look to make sure everything looks the way you intended.  If not, now is the time for us to fix it up. The figure below shows line counts and spacing.
 
 ```yml
-{% raw %}
```

Comment:
Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.